tests/helpers.py

```diff
@@ -14,12 +14,109 @@
 from tests import login_for_tests
 
 
+def get_frameworks_list_fixture_data():
+    old_gcloud_lots = [api_stubs.lot(lot_id=1, slug='saas', name='Software as a Service'),
```

Comment:
`old_gcloud_lots` and `new_gcloud_lots` should already be defined in the api_stubs file I think, so I reckon you could import those directly if you need them.

Reply:
Oh you just lift and shifted this. Nevermind then maybe 🤷‍♂️

Reply:
Yeah, I just c+p-ed this as an example. In theory this could go in `dm_test_utils` or be replaced by api stubs.